docs/source/doc-conda-requirements.yml

```diff
@@ -11,7 +11,6 @@ dependencies:
   - jinja2
   - sphinx-panels
   - sphinx-copybutton
-  - tomopy
   - pyyaml
   - ipython
```

Comment:
we can also remove ipython?

Reply:
Keeping in the meantime for use with jupyter notebook